server/prisma/schema.prisma

```diff
@@ -50,26 +50,26 @@ enum Gender {
 }
 
 model BruteBody {
-  id      Int   @id @default(autoincrement())
-  brute   Brute @relation(fields: [bruteId], references: [id])
-  bruteId Int   @unique
-  p2      Int   @default(0) // Body size (small = 0, big = 7) (male only, 0 for females)
-  p3      Int   @default(0) // Hair [0-11] 12 = no head
-  p4      Int   @default(0) // (male) Beard [0-4] 5 = nothing / (female) Front hair [0-2] 3 = nothing
-  p7      Int   @default(0) // Main clothing [0-6] 7 = naked
-  p1      Int   @default(0) // Armor variation [0-1]
-  p1a     Int   @default(0) // [0-1] 0=belt, 1=nothing
-  p1b     Int   @default(0) // [0-1] 0=roman belt, 1=nothing
-  p6      Int   @default(0) // (male) 0=shorts, 1=trousers / (female) 0=shorts, 1=nothing
-  p8      Int   @default(0) // [0-4] Supposed to be shoes but doesn't change anything ??
-  p7b     Int   @default(0) // Shoes underside (2 = visible, others = hidden)
-  p5      Int   @default(0) // 0=Nothing, 1=Shirt.
+  id      String @id @unique @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
```

Comment:
I assume @unique check in the database to be sure it's unique , It is a very low probability of do two same uuid but juste in case to be sure , With the amount of fight , brute , users etc....

I was looking for a solution that I cound use in the migration , that was the solution I found online